File: DP(Dynamic_Programming, 동적계획법)/백준_2193_이친수/이친수.java
import java.util.*;

public class Main {
    static long[] arr = new long[91];

    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        arr[1] = 1;

        for(int i = 2; i<= n; i++){
            arr[i] = arr[i-1] + arr[i-2];
        }

        System.out.println(arr[n]);
    }

}

File: DP(Dynamic_Programming, 동적계획법)/백준_2193_이친수/이친수.kt
import java.util.*
import kotlin.math.*

fun main(args: Array<String>){
    val arr = LongArray(91)
    arr[1] = 1

    val s = Scanner(System.`in`)
    val input = s.nextInt()

    for(i in 2..input){
        arr[i] = arr[i-1] + arr[i-2]
    }

    print(arr[input])

}